### ISSUE TITLE
Doc does not show a way to rename a button

### ISSUE THREAD

**Issue by FreddyCrugger** [Laravel-Backpack/Base#422]
Hello,

In Backpack 4.0, I am reading the documentation trying to work out how to rename a simple CRUD button and cannot find the info on "Rename button".
https://backpackforlaravel.com/docs/4.0/crud-buttons

I can see delete, hide but nothing about renaming an existing button, is this normal please?

Thank you,

**Comment by FreddyCrugger** [comment]
So if we pay for a premium license we do not receive any support at all? Ohhhh, that's a bit scary.

**Comment by tabacitu** [comment]
Hi @FreddyCrugger ,

Indeed, the docs don't specify how to rename a button - it's because we haven't had this request until now 😄  To change the text on a button you can:

**(Option A) Change the corresponding language file**

For example, for the Create button (```vendor/backpack/crud/src/resources/views/crud/buttons/create.blade.php```) you'll notice the actual text that's echoed is ```trans('backpack::crud.add')```. So you can create a new language file in ```resources/lang/vendor/backpack/en/crud.php``` with the following content:

```php
<?php

return [
    // Overwrite default translations
    'add' => 'Create',
];
```

This will replace the content of that text line, for the English language.

**(Option B) Change the button's blade file**

In Backpack, you can overwrite a _default_ blade file just by placing a _custom_ blade file, with the same name, in your views directory. For example, to overwrite the Create button (```vendor/backpack/crud/src/resources/views/crud/buttons/create.blade.php```) you can create a blade file in your project using the same directory structure (```resources/views/vendor/backpack/crud/buttons/create.blade.php```). Backpack will then load _your custom file_, instead of the one provided by the package. In your custom blade file, you can change whatever you want, including the text on the button:

```diff
@if ($crud->hasAccess('create'))
	<a href="{{ url($crud->route.'/create') }}" class="btn btn-primary" data-style="zoom-in"><span class="ladda-label"><i class="fa fa-plus"></i> 
-         {{ trans('backpack::crud.add') }} {{ $crud->entity_name }
+         Create {{ $crud->entity_name }
         }</span></a>
@endif
```

Hope this clears things up. Let me know if it doesn't.
Cheers!

PS. Indeed, **there's no _official_ support provided by Backpack. We are active daily on Github, email and Gitter, and try to help as much as we can, but we don't guarantee an answer to support questions.** **This is because we've had customers in the past with unreasonable expectations for support** - purchasing one $29 license then sending dozens of support requests per day, expecting Skype calls, back-and-forth on their custom code, asking us to jump into their code and help - basically requiring dozens if not hundreds of hours of support. Unfortunately we don't have the resources to handle that kind of support, or even just a few difficult customers like that, it would run us out of business. To avoid situations like that, we just call it "_no support_". **But one could say we're more active than projects that _do_ boast about offering support. Plus we have a pretty active community on StackOverflow and Gitter, with generous people who help each other out.** If "no support" is a concern to you, I recommend you look for alternative admin panels. Or shoot an email at hello@tabacitu.ro if you're willing to pay extra for support - that's something we're considering.

**Comment by FreddyCrugger** [comment]
Thank you so much for the explanation.
After playing around I found out that to rename my button I have to also rename the entity so I changed this:
$this->crud->setEntityNameStrings('New name', 'New names');

And this have change the button name to the correct name. I did not know the entity name was linked to the button name until I read your example above that showed:
`Create {{ $crud->entity_name }`

Thanks again.
